pkg/queue/rabbitmq/connection/connectionManager.go

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022-2023 Exactpro (Exactpro Systems Limited)
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connection
+
+import (
+	"fmt"
+	"github.com/rs/zerolog"
+	"os"
+)
+
+type Manager struct {
+	Publisher *Publisher
+	Consumer  *Consumer
+
+	Logger zerolog.Logger
+}
+
+func NewConnectionManager(connConfiguration Config, logger zerolog.Logger) (Manager, error) {
```

Comment:
Maybe ConnectionManager should be private?

Reply:
Let's do it. It is better to expose it in future if we need it